### PR TITLE
fix enum type op in one transaction

### PR DESCRIPTION
**Pull request by YANGGMM**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #17132 

## What this PR does / why we need it:
fix enum type op in one transaction

### CODE REVIEW

**Review by ouyuanning — APPROVED** [review]

**Review by m-schen — APPROVED** [review]

**Review by XuPeng-SH — APPROVED** [review]

**Review by daviszhen — APPROVED** [review]

**Review by qingxinhome — APPROVED** [review]

File: pkg/frontend/util.go [hunk 1]
```diff
@@ -917,6 +917,19 @@ func convertRowsIntoBatch(pool *mpool.MPool, cols []Column, rows [][]any) (*batc
 			if err != nil {
 				return nil, nil, err
 			}
+		case types.T_enum:
```
Comment (fengttt) [inline pkg/frontend/util.go:920]: look at the code, we new vData, copy into vData, then copy into vector.   Why.

extend vector and just copy into the vector directly?

Comment (YANGGMM) [reply pkg/frontend/util.go:920]: Other data types do the same thing

Comment (fengttt) [reply pkg/frontend/util.go:920]: https://github.com/matrixorigin/matrixone/issues/17155

Please create this issue yourself next time :).

**Review by fengttt — APPROVED** [review]

File: pkg/frontend/util.go (hunk 1 above)
Comment (YANGGMM) [reply pkg/frontend/util.go:920]: ok,thanks

**Pull request merged**